Thunder/utils/custom_dl.py

```
@@ -1,371 +1,60 @@
-        offset: int,
-        first_part_cut: int,
-        last_part_cut: int,
-        part_count: int,
-        chunk_size: int,
-    ) -> AsyncGenerator[bytes, None]:
-        client = self.client
-        work_loads[index] += 1
+            logger.debug(f"Error fetching message {message_id}: {e}")
+            raise FileNotFound(f"Message {message_id} not found")
+
+    async def stream_file(self, message_id: int, offset: int = 0, limit: int = 0) -> AsyncGenerator[bytes, None]:
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)